### PR TITLE
Upload skeleton blog post for GSoC and Outreachy intros

### PR DESCRIPTION
**Pull request by jennaswa**
Here is a skeleton for the GSoC and Outreachy 2024 introductory blog post. @MDAnalysis/gsoc-mentors and @micaela-matta (i.e., outreachy mentors) should please confirm that they are happy with the formatting for the skeleton.

Contributors will be asked to fill in the details, and can look at previous project announcement posts (e.g., https://github.com/MDAnalysis/MDAnalysis.github.io/blob/master/_posts/2023-05-16-gsoc-students.md) for inspiration. It is optional to add a picture. If anyone is uncomfortable with this, we can include an avatar or simply not have a photo at all.

### CODE REVIEW

**Review by richardjgowers — APPROVED** [review]

File: _posts/2024-05-27-gsoc-outreachy-students.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+You can find Valerij on Github as [talagayev](https://github.com/talagayev) and on [LinkedIn](https://www.linkedin.com/in/valerij-talagayev-260bb820b). To see the updates on the 2D visualization of small molecules project follow his [blog](https://talagayev.github.io/). 
+
+## Luna Morrow: [Extend MDAnalysis Interoperability with OpenBabel](https://summerofcode.withgoogle.com/programs/2024/projects/yLzX6MjS)
+
+<img
+src="https://avatars.githubusercontent.com/lunamorrow"
+title="Luna Morrow" alt="Luna Morrow"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+Direct interoperability between molecular dynamics software is critical for enabling collaboration, data transfer and easy use by scientists. [OpenBabel](http://openbabel.org) is a popular toolbox for chemical molecular modelling as it enables searching, file conversions, basic analysis and data storage. The ability to interconvert between OpenBabel OBMols and MDAnalysis AtomGroups will enable input and writing of over 100 chemical data file formats.
```
Comment (cbouy) [inline _posts/2024-05-27-gsoc-outreachy-students.md:47]: Very nitpicky suggestion: not sure if `data storage` makes much sense, so I'd rephrase it as such:
```suggestion
Direct interoperability between molecular dynamics software is critical for enabling collaboration, data transfer and easy use by scientists. [OpenBabel](http://openbabel.org) is a popular toolbox for chemical molecular modelling as it allows converting, searching, filtering and analyzing chemical data. The ability to interconvert between OpenBabel OBMols and MDAnalysis AtomGroups will enable input and writing of over 100 chemical data file formats.
```

File: _posts/2024-05-27-gsoc-outreachy-students.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+## Valerij Talagayev: [2D Visualization for Small Molecules](https://summerofcode.withgoogle.com/programs/2024/projects/sfy3kuqc)
+
+<img
+src="https://avatars.githubusercontent.com/talagayev"
+title="Valerij Talagayev" alt="Valerij Talagayev"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+2D Visualization of small molecules is quite a helpful way to recognize the different parts of the small molecule. The main objective of Valerij's GSoC project will revolve around the 2D visualization of small molecules through the application of the MDAnalysis RDKit converter, which would allow to convert ligands into RDKit objects and thus allow the user to look at the ligand in his MDAnalysis Universe in an easy to use interactive display in [Jupyter Notebook](https://jupyter-notebook.readthedocs.io/en/stable/).
```
Comment (cbouy) [inline _posts/2024-05-27-gsoc-outreachy-students.md:34]: suggestion: I think the first sentence is a bit redundant with the rest. I've also added a few words to make the link with RDKit clearer (i.e. we use it for its drawing code) and also how these drawing can be useful from an MD perspective:
```suggestion
The main objective of Valerij's GSoC project will revolve around the visualization of small molecules in 2D through the application of the MDAnalysis RDKit converter, which allows converting MDAnalysis atomgroups into RDKit objects and benefit from its powerful drawing code to display small molecules. These can then be annotated at the atom level with information associated with the MDAnalysis Universe or arbitrary data in an easy to use interactive display in [Jupyter Notebooks](https://jupyter-notebook.readthedocs.io/en/stable/).
```

**Review by cbouy — COMMENTED** [review]
Very minor comments from me, feel free to ignore!

File: _posts/2024-05-27-gsoc-outreachy-students.md (hunk 1 above)
Comment (jennaswa) [reply _posts/2024-05-27-gsoc-outreachy-students.md:47]: Pinging @lunamorrow to confirm approval of the change

File: _posts/2024-05-27-gsoc-outreachy-students.md (hunk 2 above)
Comment (jennaswa) [reply _posts/2024-05-27-gsoc-outreachy-students.md:34]: Pinging @talagayev to confirm approval of the change

File: _posts/2024-05-27-gsoc-outreachy-students.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jennaswa) [reply _posts/2024-05-27-gsoc-outreachy-students.md:47]: Pinging @lunamorrow as an FYI

File: _posts/2024-05-27-gsoc-outreachy-students.md (hunk 2 above)
Comment (talagayev) [reply _posts/2024-05-27-gsoc-outreachy-students.md:34]: looks good to me, thanks @jennaswa @cbouy :)

**Review by hmacdope — APPROVED** [review]

File: _posts/2024-05-27-gsoc-outreachy-students.md (hunk 1 above)
Comment (lunamorrow) [reply _posts/2024-05-27-gsoc-outreachy-students.md]: Are we able to say "[OpenBabel](https://openbabel.org) is a popular toolbox for chemical molecular modelling as it enables conversion, searching, filtering and analysis of chemical data." instead? I agree about removing the data storage aspect but I'm getting a bit nit-picky myself about the grammar.

Comment (cbouy) [reply _posts/2024-05-27-gsoc-outreachy-students.md]: Absolutely fine by me, never trust me for grammar 😌 I switched to use _allow_ since _enable_ is already in the previous and next sentences but no big deal

**Review by yuxuanzhuang — APPROVED** [review]

File: _posts/2024-05-27-gsoc-outreachy-students.md (hunk 1 above)
Comment (lunamorrow) [reply _posts/2024-05-27-gsoc-outreachy-students.md]: Easy! Do I need to checkout this PR and make that change on my end? I am not familiar with how we normally apply revisions? @cbouy @jennaswa

Comment (cbouy) [reply _posts/2024-05-27-gsoc-outreachy-students.md]: In the code section of the PR you can click on the + next to a line to add a comment and there should be a button to create a suggestion there

Comment (jennaswa) [reply _posts/2024-05-27-gsoc-outreachy-students.md]: Since this was the only outstanding comment, I went ahead and included @lunamorrow's sentence, but switched out "enables" with "allows" - hope that is okay!

Comment (lunamorrow) [reply _posts/2024-05-27-gsoc-outreachy-students.md]: > In the code section of the PR you can click on the + next to a line to add a comment and there should be a button to create a suggestion there

Thanks! Great to know for next time :)

> Since this was the only outstanding comment, I went ahead and included @lunamorrow's sentence, but switched out "enables" with "allows" - hope that is okay!

That is perfect, thank you for making that adjustment @jennaswa :)
